prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -500,7 +500,7 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
             keyManagerFactory.init(keyStore, jksPasscode)
             // create the sslContext, type TLS, with the keyManager
             // note the ktor sample code uses a TrustManger instead, but that fails with the NY cert file
-            val sslContext = SSLContext.getInstance("TLS")
+            val sslContext = SSLContext.getInstance("TLSv1.2")
```

Comment:
1.3 would be a better long term choice. https://github.com/CDCgov/prime-reportstream/pull/9515/files

I'm told that this code affects all REST partners.

Reply:
and NY is capable of 1.3
https://www.ssllabs.com/ssltest/analyze.html?d=api2.health.ny.gov&hideResults=on

Reply:
It only affects NY as the `getSslContext` function only runs given the receiver has a configured JKS for client certificate auth. Our other REST partners (OK, WA) don't use that so this code does not run.

Reply:
In that case, this should be"TLSv1.3".

Reply:
Sounds good but let me double check that this works with JDK 11.

Reply:
worked fine

![Screenshot 2023-06-27 at 2 40 51 PM](https://github.com/CDCgov/prime-reportstream/assets/1209453/e71f3b37-2094-43c4-a1f5-0db303e244a7)